scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -104,9 +106,44 @@ class RedundantBraces(implicit ctx: RewriteCtx) extends RewriteSession {
       case t: Term.Interpolate if settings.stringInterpolation =>
         processInterpolation(t)
 
+      case i: Importer =>
+        processImport(i)
```

Comment:
would this be more readable:
```
  case Importer(_, List(importee)) => processSingleImport(importee)
```

Reply:
Good idea, changed and added a condition for the dialect.